### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by rogeruiz** [rogeruiz/tick#5]
## Detailed Description
Because of the dependency on [Diesel](http://diesel.rs), the build step will need to leverage [Syntex](https://github.com/serde-rs/syntex) to properly build and test the final binary.

## Context
Tests are currently broken so it's a good idea to just write a basic test and fix the compilation steps.

## Possible Implementation
Follow the guide [at this point](http://diesel.rs/guides/getting-started/#on-stable-rust) to leverage Syntex.

## Your Environment
* Travis-CI

**Comment by rogeruiz** [comment]
Looks like I'm running into issues with a few things. First it was the fact that `stable` is unsupported in the codebase currently, but when I solved that with switching to `nightly` I ran into the problem that there's not a `DATABASE_URL` environment variable nor a sample database used during the Travis build.

**Comment by rogeruiz** [comment]
Now that there is a release :tada: that means I can focus on writing tests for all the funky panics I caught while I was debugging the release.

**Comment by rogeruiz** [comment]
According to @steveklabnik in the `#rust-beginners` channel in IRC.

```txt
10:14 <~steveklabnik> rsr: https://github.com/rogeruiz/tick/blob/master/.travis.yml <- you never set up your database
10:14 < rsr> oh doh! I thought so. so I need to run the migration. ugh. I just made an issue for that. hah
             https://github.com/rogeruiz/tick/issues/7
```

**Comment by rogeruiz** [comment]
Fixed the build on Travis, but there are still no tests. Looks like I should be splitting things up into a `main.rs` for the clap implementation and `lib.rs` for the actual library bits.

**Comment by rogeruiz** [comment]
Closed by #14
